apps/Gruntfile.js

```diff
@@ -359,53 +359,53 @@ module.exports = function (grunt) {
+    'shared/_district_dropdown':    './src/sites/studio/pages/shared/_district_dropdown.js',
     'levelbuilder':                 './src/sites/studio/pages/levelbuilder.js',
-    'levelbuilder_applab':          './src/sites/studio/pages/levelbuilder_applab.js',
-    'levelbuilder_edit_script':     './src/sites/studio/pages/levelbuilder_edit_script.js',
-    'levelbuilder_gamelab':         './src/sites/studio/pages/levelbuilder_gamelab.js',
-    'levelbuilder_markdown':        './src/sites/studio/pages/levelbuilder_markdown.js',
-    'levelbuilder_studio':          './src/sites/studio/pages/levelbuilder_studio.js',
-    'levels/contract_match':        './src/sites/studio/pages/levels/contract_match.jsx',
+    'levels/editors/_applab':       './src/sites/studio/pages/levels/editors/_applab.js',
+    'scripts/_form':                './src/sites/studio/pages/scripts/_form.js',
+    'levels/editors/_gamelab':      './src/sites/studio/pages/levels/editors/_gamelab.js',
+    'levels/editors/_dsl':          './src/sites/studio/pages/levels/editors/_dsl.js',
+    'levels/editors/_studio':       './src/sites/studio/pages/levels/editors/_studio.js',
+    'levels/_contract_match':       './src/sites/studio/pages/levels/_contract_match.jsx',
     'levels/dashboardDialogHelper': './src/sites/studio/pages/levels/dashboardDialogHelper.js',
-    'levels/external':              './src/sites/studio/pages/levels/external.js',
-    'levels/levelGroup':            './src/sites/studio/pages/levels/levelGroup.js',
+    'levels/_external':             './src/sites/studio/pages/levels/_external.js',
+    'levels/_level_group':          './src/sites/studio/pages/levels/_level_group.js',
     'levels/multi':                 './src/sites/studio/pages/levels/multi.js',
-    'levels/textMatch':             './src/sites/studio/pages/levels/textMatch.js',
-    'levels/widget':                './src/sites/studio/pages/levels/widget.js',
-    'signup':                       './src/sites/studio/pages/signup.js',
-    'raceInterstitial':             './src/sites/studio/pages/raceInterstitial.js',
-    'termsInterstitial':            './src/sites/studio/pages/termsInterstitial.js',
-    'makerlab/setupPage':           './src/sites/studio/pages/setupMakerlab.js',
-    'scriptOverview':               './src/sites/studio/pages/scriptOverview.js'
+    'levels/_text_match':           './src/sites/studio/pages/levels/_text_match.js',
+    'levels/_widget':               './src/sites/studio/pages/levels/_widget.js',
+    'devise/registrations/new':     './src/sites/studio/pages/devise/registrations/new.js',
+    'layouts/_race_interstitial':   './src/sites/studio/pages/layouts/_race_interstitial.js',
+    'layouts/_terms_interstitial':  './src/sites/studio/pages/layouts/_terms_interstitial.js',
+    'maker/setup':                  './src/sites/studio/pages/maker/setup.js',
+    'scripts/show':                 './src/sites/studio/pages/scripts/show.js'
```

Comment:
fortunately, I think build times do not correlate linearly with number of entry points but rather number of total modules, so as long as all these entry points are just reusing the same modules, it hopefully won't get a lot slower.

But otherwise, I am always excited about improving build performance!